src/Test/WordPress/HooksTest.php

```diff
@@ -42,9 +42,10 @@ public function setup()
         $this->mockDefaultIntegration = new DefaultIntegration($this->mockConfig, $this->mockWordPressAPI, $this->mockDataStore, $this->mockLogger);
 
         $this->hooks = $this->getMockBuilder('\CF\WordPress\Hooks')
-            ->setMethods(array('setHooks'))
+            ->disableOriginalConstructor()
+            ->setMethods(array('__construct')) // This is a hack to make the tests work
```

Comment:
`->disableOriginalConstructor()` disables the original constructor and then you mock one with `->setMethods(array('__construct'))`.  Do you need the constructor?  If so PHPUnit allows you to mock with it.